x-pack/plugins/case/server/routes/api/cases/comments/delete_comment.ts

```diff
@@ -23,14 +26,22 @@ export function initDeleteCommentApi({ caseService, router }: RouteDeps) {
     async (context, request, response) => {
       try {
         const client = context.core.savedObjects.client;
-        const myCase = await caseService.getCase({
+        const { username, full_name, email } = await caseService.getUser({ request, response });
+        const deleteDate = new Date().toISOString();
+
+        const myComment = await caseService.getComment({
           client: context.core.savedObjects.client,
```

Comment:
nit: im gonna stop pointing it out everywhere but can we just be consistent about this client variable?

Reply:
Yes, I did not mean to not be consistent, I will go over and fix them